nautobot/extras/views.py

```diff
@@ -1040,8 +1040,7 @@ def extra_context(self):
         }
 
 
-# 2.0 TODO: this should really be "JobRunView"
-class JobView(ObjectPermissionRequiredMixin, View):
+class JobRunView(ObjectPermissionRequiredMixin, View):
```

Comment:
Should be able to change the implementation of this function to no longer need to instantiate the `job_class` to a `job_instance` now that we can directly call `job_class.as_form_class()`/`job_class.as_form()`. That was part of the reason why we wanted to make these into classmethods.


Should be able to similarly change the API `JobViewSet.run()` to do `job_class.validate_data(...)` instead of `job_class().validate_data(...)`.